stream/producer_chain.go

```diff
@@ -151,20 +150,15 @@ func (p *producerChainContainer) getIndex() error {
 	p.nodeIndex = nodeIndex
 	p.nodeIndex.Instance = p.nodeinstance
 	p.nodeIndex.Topic = p.topic
-	p.sc.Log.Info("starting processing %d", p.nodeIndex.Idx)
+	p.sc.Log.Info("starting processing", zap.Uint64("node index", p.nodeIndex.Idx))
 	return nil
 }
 
 func (p *producerChainContainer) ProcessNextMessage() error {
-	containerRangeArgs := &indexer.GetContainerRangeArgs{
-		StartIndex: json.Uint64(p.nodeIndex.Idx),
-		NumToFetch: json.Uint64(MaxTxRead),
-		Encoding:   formatting.Hex,
-	}
 	ctx, cancelCtx := context.WithTimeout(context.Background(), IndexerTimeout)
 	defer cancelCtx()
 
-	containers, err := p.nodeIndexer.GetContainerRange(ctx, containerRangeArgs)
+	containers, err := p.nodeIndexer.GetContainerRange(ctx, p.nodeIndex.Idx, MaxTxRead)
```

Comment:
Does this return hex by default now?  Just double checking the hex/non-hex won't break anything

Reply:
This function doesn't expose any formatted bytes.

Reply:
So this function is no longer returning formatted data?  If so, can you direct me to the avalancheGo code that handled formatting?

Reply:
Looks like this function still returns [formatted data](https://github.com/ava-labs/avalanchego/blob/master/indexer/client.go#L71) with bytes in [hex](https://github.com/ava-labs/avalanchego/blob/master/indexer/client.go#L59) format by default.

Reply:
This function returns [decoded data](https://github.com/ava-labs/avalanchego/blob/master/indexer/client.go#L67) without exposing any internal formats.